app/views/admin/events/show.html.haml

```diff
@@ -13,7 +13,9 @@
 
   .tab-content
     #proposal-content.tab-pane.active
+      Karate Kid Before
```

Comment:
Oh my... I'll remove that and push again. :-) haha that was me trying to learn where the partial started and ended.  Just random text that I knew would be unique within the DOM.